content/home/milestones/heading/milestones.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "milestones-heading"
 heading: Roadmap Milestones to Production
-subheading: Tardigrade is currently in the Pioneer 1 phase of our beta. Those who sign up for our waitlist can also participate in our Beta ahead of production later this year.
----
\ No newline at end of file
+subheading: Tardigrade is currently in the Pioneer 2 phase of our beta. Those who sign up for our waitlist can also participate in our Beta ahead of production later this year.
```

Comment:
is it later this year, or early next year? is there an official date we can give?

Reply:
hmm Taylor reviewed and said all the copy was correct, but this is a good question